requirements.txt

```diff
@@ -4,3 +4,7 @@ SQLAlchemy~=2.0.41
 
 # SQLAlchemy requires manual installation of database drivers
 psycopg2-binary~=2.9.10
+
+# dbt for schema management
+dbt-core~=1.8.0
```

Comment:
This can be removed, as it will be installed as a dependency of `dbt-postgres`. We don’t install it explicitly in downstream project